cardano-node/src/Cardano/Node/Types.hs

```diff
@@ -282,17 +142,17 @@ data NodeAlonzoProtocolConfiguration =
 
 data NodeByronProtocolConfiguration =
      NodeByronProtocolConfiguration {
-       npcByronGenesisFile         :: !GenesisFile
-     , npcByronGenesisFileHash     :: !(Maybe GenesisHash)
-     , npcByronReqNetworkMagic     :: !RequiresNetworkMagic
-     , npcByronPbftSignatureThresh :: !(Maybe Double)
+       npcByronGenesisFile                   :: !GenesisFile
```

Comment:
A perfect example of my we enforce _not_ aligning code. It introduces unnecessary diff.

Reply:
I don't see what you mean